app/_hub/kong-inc/ai-proxy/_changelog.md

```diff
@@ -1,5 +1,25 @@
 ## Changelog
 
+### {{site.base_gateway}} 3.9.x
+* Disabled the HTTP/2 ALPN handshake for connections on routes configured with AI Proxy.
+[#13735](https://github.com/Kong/kong/issues/13735)
+* Fixed an issue where tools (function) calls to Anthropic would return empty results.
+[#13760](https://github.com/Kong/kong/issues/13760)
+* Fixed an issue where tools (function) calls to Bedrock would return empty results.
+[#13760](https://github.com/Kong/kong/issues/13760)
+* Fixed an issue where Bedrock Guardrail config was ignored.
+[#13760](https://github.com/Kong/kong/issues/13760)
+* Fixed an issue where tools (function) calls to Cohere would return empty results.
+[#13760](https://github.com/Kong/kong/issues/13760)
+* Fixed an issue where the Gemini provider would return an error if content safety failed in AI Proxy.
+[#13760](https://github.com/Kong/kong/issues/13760)
+* Fixed an issue where tools (function) calls to Gemini (or via Vertex) would return empty results.
+[#13760](https://github.com/Kong/kong/issues/13760)
+* Fixed an issue where AI Transformer plugins always returned a 404 error when using Google One Gemini subscriptions.
```

Comment:
Nice catch - this is under the wrong plugin. it was categorized under AI Proxy by the dev, but that's because he fixed another issue in the same PR for that plugin.